ops/op2/dispatch_fast.rs

```diff
@@ -0,0 +1,195 @@
+      FastValue::Pointer => quote!(*mut ::std::ffi::c_void),
+      FastValue::V8Value => quote!(v8::Local<v8::Value>),
+      FastValue::Uint8Array
+      | FastValue::Uint32Array
+      | FastValue::Float64Array
+      | FastValue::SeqOneByteString => unreachable!(),
+    }
+  }
+
+  /// Quote fast value type's variant.
+  fn quote_ctype(&self) -> TokenStream {
+    match &self {
+      FastValue::Void => quote!(CType::Void),
+      FastValue::Bool => quote!(CType::Bool),
+      FastValue::U32 => quote!(CType::Uint32),
+      FastValue::I32 => quote!(CType::Int32),
+      FastValue::U64 => quote!(CType::Uint64),
+      FastValue::I64 => quote!(CType::Int64),
+      FastValue::F32 => quote!(CType::Float32),
+      FastValue::F64 => quote!(CType::Float64),
+      FastValue::Pointer => quote!(CType::Pointer),
+      FastValue::V8Value => quote!(CType::V8Value),
+      FastValue::Uint8Array => unreachable!(),
+      FastValue::Uint32Array => unreachable!(),
+      FastValue::Float64Array => unreachable!(),
+      FastValue::SeqOneByteString => quote!(CType::SeqOneByteString),
+    }
+  }
+
+  /// Quote fast value type's variant.
+  fn quote_type(&self) -> TokenStream {
+    match &self {
+      FastValue::Void => quote!(Type::Void),
+      FastValue::Bool => quote!(Type::Bool),
+      FastValue::U32 => quote!(Type::Uint32),
+      FastValue::I32 => quote!(Type::Int32),
+      FastValue::U64 => quote!(Type::Uint64),
+      FastValue::I64 => quote!(Type::Int64),
+      FastValue::F32 => quote!(Type::Float32),
+      FastValue::F64 => quote!(Type::Float64),
+      FastValue::Pointer => quote!(Type::Pointer),
+      FastValue::V8Value => quote!(Type::V8Value),
+      FastValue::Uint8Array => quote!(Type::TypedArray(CType::Uint8)),
+      FastValue::Uint32Array => quote!(Type::TypedArray(CType::Uint32)),
+      FastValue::Float64Array => quote!(Type::TypedArray(CType::Float64)),
+      FastValue::SeqOneByteString => quote!(Type::SeqOneByteString),
+    }
+  }
+}
+
+pub fn generate_dispatch_fast(
+  generator_state: &mut GeneratorState,
+  signature: &ParsedSignature,
+) -> Result<Option<(TokenStream, TokenStream)>, V8MappingError> {
+  let mut inputs = vec![];
+  for arg in &signature.args {
+    let fv = match arg {
+      Arg::OptionNumeric(_) | Arg::SerdeV8(_) => return Ok(None),
+      Arg::Numeric(NumericArg::bool) => FastValue::Bool,
+      Arg::Numeric(NumericArg::u32)
+      | Arg::Numeric(NumericArg::u16)
+      | Arg::Numeric(NumericArg::u8) => FastValue::U32,
+      Arg::Numeric(NumericArg::i32)
+      | Arg::Numeric(NumericArg::i16)
+      | Arg::Numeric(NumericArg::i8)
+      | Arg::Numeric(NumericArg::__SMI__) => FastValue::I32,
+      Arg::Numeric(NumericArg::u64) | Arg::Numeric(NumericArg::usize) => {
+        FastValue::U64
+      }
+      Arg::Numeric(NumericArg::i64) | Arg::Numeric(NumericArg::isize) => {
+        FastValue::I64
+      }
+      // TODO(mmastrac)
+      Arg::Special(_) => return Ok(None),
```

Comment:
Will this fallback to slow path? If not, maybe use `unreachable!()` or `todo!()` for now?

Reply:
I'll remove it for now -- I don't think it needs to be here for this pass.